nestkernel/simulation_manager.cpp

```diff
@@ -813,27 +842,6 @@ nest::SimulationManager::update_()
 }
```

Comment:
Additional comment reg. line 791 in the OLD code (in function update_()): I wonder if this part of the code outside of the update loop will work properly if running directly again without a cleanup/prepare cycle. It seems to be related to structural plasticity and updates synaptic elements.
@sdiazpier : May I ask you for help regarding this question?